### DISCUSSION TITLE
Flashed 7510 not booting

### DISCUSSION THREAD

**Discussion by pwFoo** [Freetz-NG/freetz-ng#828] [Allgemein] [answered]
Hi,
I created to flash my new 7510 with freetz-ng. Build and flash an image works fine, but also a minimal (just box selected) image fails to boot.
Bootloader is available and 4 pings replied, but than all leds off and nothing happens. Re-flash works, but same problem with different images (last one was minimal...).

Any idea what's happening?

**Comment by pwFoo** [comment]
Tested with latest commit today, but same error. Still not able to boot minimal freetz firmware with 7510

**Comment by fda77** [comment]
As you could read in another diskcussion (where you postet too) it worked for @freetz-ng-mod

**Comment by pwFoo** [comment]
Hi @fda77,

thanks for reply to my problem. Don't know what I should say. I tried it many times and cleaned up some times... Build works fine, but Image isn't bootable...
LEDs off and nothing happens. So it seems to be broken...
Sounds the same as #825. 

Frustrating start with freetz-ng. Found any helping issues, discussions or documentation / wiki... 

Flashing the avm orig image works fine... But missing primary shell and tcpdump with the orig firmware

**Comment by fda77** [comment]
You could try the revision the other used

**Comment by pwFoo** [comment, marked as answer]
Hi,
I flashed original firmware, built a new one from latest commit and now same way works fine... So 7510 is flashed now, but with strange behaviour...

Which load is "normal" (EDIT: for a freetz'ed FritzBox)?
```
root@fritz:/var/mod/root# uptime
 20:15:24 up 31 min,  load average: 244.15, 82.18, 32.81

root@fritz:/var/mod/root# uptime
 20:16:52 up 33 min,  load average: 59.58, 62.07, 30.03

root@fritz:/var/mod/root# uptime
 20:18:47 up 35 min,  load average: 112.03, 90.54, 44.51

root@fritz:/var/mod/root# uptime
 20:21:05 up 37 min,  load average: 11.54, 56.96, 38.39
```

Used for WLAN and DECT inside of my lan (ip client). Don't know about performance short time I switched from 7530ax to 7510 here... 

Load now reduced uptime 45min:
```
Load average: 0.38 9.16 21.23 4/216 7532
```

**Reply by fda77** [reply]
load 0.00 = 0%
load 1.00 = 100% with 1 cpu core
load 4.00 = 4cores @ 100% load

... so > 100.0 is some bad avm hacking

maybe you see in syslog some hints

**Comment by pwFoo** [comment]
At the moment it's <0,2... 
Strange... Box boot is booting. Would close here for now. Thanks!
